src/pages/DashboardPage.js

```diff
@@ -1,37 +1,60 @@
 import dayjs from 'dayjs';
 import { transparentize } from 'polished';
 import React, { useEffect, useState } from 'react';
+import { DollarSign, FileText } from 'react-feather';
 import { withRouter } from 'react-router-dom';
 import { useMedia } from 'react-use';
 import styled from 'styled-components';
 
 import { TYPE, ThemedBackground } from '../Theme';
 import { PageWrapper, ContentWrapper } from '../components';
 import { AutoColumn } from '../components/Column';
+import ComulativeNetworkDataCard from '../components/ComulativeNetworkDataCard';
 import LocalLoader from '../components/LocalLoader';
 import Panel from '../components/Panel';
 import StackedChart from '../components/StackedChart';
-import { useDashboardChartData } from '../contexts/Dashboard';
+import { SupportedNetwork } from '../constants';
+import { useDashboardChartData, useDashboardComulativeData } from '../contexts/Dashboard';
+import { formattedNum } from '../utils';
 
 const GridRow = styled.div`
   display: grid;
-  width: 100%;
-  grid-template-columns: 1fr 1fr;
-  column-gap: 6px;
-  align-items: start;
-  justify-content: space-between;
+  grid-template-columns: 3fr 1.5fr;
+  column-gap: 16px;
+  row-gap: 16px;
 `;
 
+const GridChart = styled.div`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 416px 416px;
+  row-gap: 16px;
+`;
+
+const GridCard = styled.div`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  row-gap: 16px;
+`;
+
+const PanelLoaderWrapper = ({ isLoading, children }) => <Panel>{isLoading ? <LocalLoader /> : children}</Panel>;
+
 const DashboardPage = () => {
   const chartData = useDashboardChartData();
+  const comulativeData = useDashboardComulativeData();
   const [formattedLiquidityData, setFormattedLiquidityData] = useState([]);
   const [formattedVolumeData, setFormattedVolumeData] = useState([]);
+  const [formattedTotalTrades, setFormattedTotalTrades] = useState([]);
+  const [formattedTotalVolume, setFormattedTotalVolume] = useState([]);
```

Comment:
use a single state and use two keys in place of `value` like `trades` and `volumes`